zone/trap.cpp

```diff
@@ -366,21 +366,66 @@ void EntityList::UpdateAllTraps(bool respawn, bool repopnow)
 
 void EntityList::GetTrapInfo(Client* client)
 {
-	uint8 count = 0;
-	auto it = trap_list.begin();
-	while (it != trap_list.end())
-	{
-		Trap* cur = it->second;
-		if (cur->IsTrap())
-		{
-			bool isset = (cur->chkarea_timer.Enabled() && !cur->reset_timer.Enabled());
-			client->Message(Chat::Default, " Trap: (%d) found at %0.2f,%0.2f,%0.2f. Times Triggered: %d Is Active: %d Group: %d Message: %s", cur->trap_id, cur->m_Position.x, cur->m_Position.y, cur->m_Position.z, cur->times_triggered, isset, cur->group, cur->message.c_str());
-			++count;
+	uint32 trap_count = 0;
+	uint32 trap_number = 1;
+
+	for (const auto& trap : trap_list) {
+		auto current_trap = trap.second;
```

Comment:
Can just use `t` in place of `current_trap` to simplify the receiver 

There are 13 calls to the receiver below should shorten things up

Reply:
👍